src/sentry/utils/snuba.py

```diff
@@ -453,8 +453,9 @@ def transform_aliases_and_query(skip_conditions=False, **kwargs):
         aggregation[1] = get_snuba_column_name(aggregation[1])
 
     for (col, _value) in six.iteritems(filter_keys):
-        name = get_snuba_column_name(col)
-        filter_keys[name] = filter_keys.pop(col)
+        if not skip_conditions:
```

Comment:
@markstory I think it might be a good idea to wrap this if condition on the entire for loop, so that the iteration can be skipped altogether.